src/Microsoft.Azure.SignalR/Utilities/AuthorizeHelper.cs

```diff
@@ -16,7 +16,7 @@ internal static class AuthorizeHelper
     {
         public static async Task<bool> AuthorizeAsync(HttpContext context, IList<IAuthorizeData> policies)
         {
-            if (policies != null && policies.Count == 0)
+            if (policies == null || policies.Count == 0)
```

Comment:
Hmm, this change looks suspicious. Why change this now?

Reply:
in 3.0, users might still call UseAzureSignalR() and will get null policies (https://github.com/Azure/azure-signalr/blob/corestartup/src/Microsoft.Azure.SignalR/ServiceRouteBuilder.cs#L47), so null should be also regarded as success?
Normally IAuthorizeData built from Hub class wouldn't be null.

Reply:
If you call UseAzureSignalR in 3.0 it should work exactly like it did in 2.x

Reply:
Right. But `AuthorizationMiddleware` is added in Core 3.0 which to reduce the effect in our SDK. Should it be skipped as here I checks (https://github.com/Azure/azure-signalr/blob/corestartup/src/Microsoft.Azure.SignalR/Utilities/AuthorizeHelper.cs#L73-#L76)? 
Or just let `UseAzureSignalR()` do the authorization build and check in 3.0?

Reply:
I think it should just do what it did before. It means there’ll be a slight performance hit but it’ll make upgrades work. The worst thing we could do is remove the check then have apps upgrade and result in less secure code by default

Reply:
Make sense. Will update.